packages/fx-core/src/core/middleware/projectMigratorV3.ts

```diff
@@ -192,12 +195,21 @@ export async function generateSettingsJson(context: MigrationContext): Promise<v
 }
 
 export async function generateAppYml(context: MigrationContext): Promise<void> {
+  const inputs: Inputs = context.arguments[context.arguments.length - 1];
+  const bicepFilePath =
+    inputs.platform === Platform.VS
+      ? Constants.vsProvisionBicepPath
+      : Constants.vscodeProvisionBicepPath;
   const bicepContent: string = await fs.readFile(
-    path.join(context.projectPath, Constants.provisionBicepPath),
+    path.join(context.projectPath, bicepFilePath),
```

Comment:
so this function: [readBicepContent](https://github.com/OfficeDev/TeamsFx/blob/dev/packages/fx-core/src/core/middleware/utils/v3MigrationUtils.ts#:~:text=export%20function%20readBicepContent(context%3A%20MigrationContext)%3A%20any%20%7B) should also be changed according to platform, right?

Reply:
or could you please change the code of readBicepContent() in utils, as this is a common function that will be often refered?

Reply:
thanks, updated